<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="direction_net_weight_path" default="/home/amsl/catkin_ws/src/vision-based_navigation/dnn_models/directionnet/weights/without_center_perimeter_full_1222/best_acc.pt"/>
    <arg name="orientation_net_weight_path" default="/home/amsl/catkin_ws/src/vision-based_navigation/dnn_models/orientationnet/weights/without_center_perimeter_full_1222/best_acc.pt"/>

    <arg name="bagfiles_dir" default="/home/amsl/bag/abstrelposnet/dkan_perimeter/for_reference"/>
    <arg name="map_save_path" default="/home/amsl/catkin_ws/src/vision-based_navigation/relative_navigator/topological_maps/dkan_perimeter_test.pkl"/>


    <!-- <node pkg="relative_navigator" type="topological_mapper_node.py" name="topological_mapper" ns="relative_navigator" output="screen" unless="$(arg use_timenet)"> -->
    <node pkg="relative_navigator" type="topological_mapper_node.py" name="topological_mapper" output="screen">
        <rosparam subst_value="True">
            direction_net_weight_path: $(arg direction_net_weight_path)
            orientation_net_weight_path: $(arg orientation_net_weight_path)
            bagfiles_dir: $(arg bagfiles_dir)
            map_save_path: $(arg map_save_path)
        </rosparam>
    </node>
</launch>
